src/game/client/shadow/w3dbuffermanager.cpp

```diff
@@ -0,0 +1,475 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+
+#include "w3dbuffermanager.h"
+#include "dx8indexbuffer.h"
+#include "dx8vertexbuffer.h"
+
+// TODO need general cleanup advice
+// TODO what is going on with count
+
+static int FVFTypeIndexList[W3DBufferManager::MAX_FVF] = {
```

Comment:
Could we introduce an alias for this specific `int` and in places it is used? This way it would be easier to see what this integer refers to (D3DFVF mask).